components/chat/messages.md

```diff
@@ -179,6 +243,145 @@ Control the width behavior of chat messages using the `MessageWidthMode` paramet
+            </ChatFileActions>
+        </ChatAuthorMessageSettings>
+
+        <ChatReceiverMessageSettings EnableMessageCollapse="false"
+                                     MessageWidthMode="@MessageWidthMode.Standard">
+            <ChatMessageContextMenuActions>
+                <ChatMessageContextMenuAction Name="Reply" Text="Reply" Icon="@SvgIcon.Undo" />
+                <ChatMessageContextMenuAction Name="Forward" Text="Forward" Icon="@SvgIcon.Forward" />
+            </ChatMessageContextMenuActions>
+            <ChatMessageToolbarActions>
+                <ChatMessageToolbarAction Icon="@SvgIcon.Heart" OnClick="@ReactToMessage" Text="Like" />
+            </ChatMessageToolbarActions>
+            <ChatFileActions>
+                <ChatFileAction Name="Download" Text="Download Shared File" />
+            </ChatFileActions>
+        </ChatReceiverMessageSettings>
+    </ChatSettings>
+</TelerikChat>
+
+@code {
+    private TelerikChat<ChatMessage> ChatRef { get; set; }
+    private List<ChatMessage> ChatData { get; set; } = new()
+    {
+        new ChatMessage()
+        {
+            Id = "1",
+            AuthorId = "support",
+            Content = "Hello! How can I assist you today?",
+            Timestamp = DateTime.Now.AddMinutes(-10)
+        },
+        new ChatMessage()
+        {
+            Id = "2",
+            AuthorId = "user1",
+            Content = "Hi, I have a question about the new features.",
+            Timestamp = DateTime.Now.AddMinutes(-5)
+        }
+    };
+    private string CurrentUserId { get; set; } = "user1";
+
+    private void OnChatSendMessage(ChatSendMessageEventArgs args)
+    {
+        var newMessage = new ChatMessage
+        {
+            Id = Guid.NewGuid().ToString(),
+            Content = args.Message,
+            AuthorId = CurrentUserId,
+            Timestamp = DateTime.Now
+        };
+
+        ChatData.Add(newMessage);
+    }
+
+    private void PinMessage(ChatMessageActionClickEventArgs args)
+    {
+        var message = ChatData.FirstOrDefault(m => m.Id == args.MessageId);
+        if (message != null)
+        {
+            message.IsPinned = true;
+            ChatRef?.Refresh();
+        }
+    }
+
+    private void ReactToMessage(ChatMessageActionClickEventArgs args)
+    {
+        Console.WriteLine($"Liked message: {args.MessageId}");
+    }
+
+    public class ChatMessage
+    {
+        public string Id { get; set; }
+        public string AuthorId { get; set; }
+        public string Content { get; set; }
+        public bool IsPinned { get; set; }
+        public DateTime Timestamp { get; set; }
+    }
+}
+````
+
+Available settings for both `ChatAuthorMessageSettings` and `ChatReceiverMessageSettings`:
+
+* `EnableMessageCollapse` - Enables the collapse functionality for long messages
+* `MessageWidthMode` - Controls message width (`Standard` or `Full`)
+* `ChatMessageContextMenuActions` - Define context menu actions for right-click interactions
```

Comment:
Ideally, all bullets should follow a unified structure. If the previous were "enables" and "controls", the remaining should be "defines".